images/miq-app/Dockerfile

```diff
@@ -82,14 +82,14 @@ WORKDIR ${APP_ROOT}
 RUN source /etc/default/evm && \
     export RAILS_USE_MEMORY_STORE="true" && \
     npm install bower yarn -g && \
-    gem install bundler -v 1.15.4 && \
+    gem install bundler -v ">=1.16.2" && \
```

Comment:
Not a big deal.. but where did this `1.16.2` come from? In appliance, we have `gem install bundler -v ">=1.8.4"`

Reply:
Well 1.15 was causing a failure to resolve so I figured it should be updated. @bdunne also mentioned that 1.16.1 had some problem, but I can't remember what it was.

The appliance should probably be updated too.

Reply:
Okay, but the ManageIQ appliance is at `1.16.5`

Reply:
It's not a problem to use 1.16 upstream as I don't think we really depends on bundler version for our functionality and this is how it has always been in master (now hammer).  But if we require `1.16.2` to be minimum (in Gemfile), that will break the downstream build...

Reply:
Ah okay, I didn't realize that was a response to @bdunne 's comment rather than the change on this line.

Reply:
oh sorry, that comment wasn't clear...